cmd/drive/main.go

```diff
@@ -1562,13 +1562,15 @@ func (cmd *unshareCmd) Run(args []string, definedFlags map[string]*flag.Flag) {
 }
 
 type moveCmd struct {
-	Quiet *bool `json:"quiet"`
-	ById  *bool `json:"by-id"`
+	Quiet      *bool `json:"quiet"`
+	ById       *bool `json:"by-id"`
+	CopyLinks  *bool `json:"copy-links"`
```

Comment:
Could we make this attribute `keep-parent` KeepParent, instead of `copy-links`. CopyLinks IMO doesn't portray the message that we are trying to preserve the original parent.